search/search_repository_blackbox_test.go

```diff
@@ -69,6 +69,76 @@ func (s *searchRepositoryBlackboxTest) getTestFixture() *tf.TestFixture {
 	)
 }
 
+func (s *searchRepositoryBlackboxTest) TestSearchWithChildIterationWorkItems() {
```

Comment:
@baijum Can you please add a few tests for a combination of `iterations` and `space/area/workitem`?
Also please add a test case for `child-iterations` set to `true` and the filter expression doesn't have any iteration.

Reply:
There is a test for the second suggestion here, line no. 102

Reply:
I have added one test with space. 2b3e231b52f21a043631598018704491e39f6956

Reply:
>There is a test for the second suggestion here, line no. 102

@baijum I meant the query has `child-iteration=true` and there are no `iterations` in the query.
Something like `space=xyz && child-iterations=true`

Reply:
Updated with space: ba3da4d2b645352a009fa3cef25da8caea33b877

Reply:
Fixed in 2f61ed4e2d6a2449865b4434370d7e22fee090ec